### PR TITLE
[DELIVERABLE 2] SRS: All my sections  #66 #68 #55 #58

### LINKED ISSUE #66: Usability and Humanity Requirements (hard)

### PR DESCRIPTION
**Pull request by mmyaaaaa**
## Description
Documentation 
Closes #66 #68 #55 #58 

## How Has This Been Tested?
None its documentation

### CODE REVIEW

**Review by Ayushi1972 — APPROVED** [review]

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -142,31 +189,129 @@ \subsection{Style Requirements}
...
+  \end{itemize}
+  \item \textbf{Requirement:} User guide must be available in French and English.
+  \begin{itemize}[label={}]
+      \item \textbf{Fit Criteria:} French and english installation and use instructions available.
+  \end{itemize} 
+\end{enumerate}
 \subsection{Learning Requirements}
-\lips
+\begin{enumerate}
+  \item \textbf{Requirement:} The tool must have an availible youtube video demonstrating installation.
```
Comment (Sevhena) [inline docs/SRS/SRS.tex:215]: We actually gonna do this?

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -215,25 +360,55 @@ \section{Open Issues}
...
+  \item \textbf{Rope:} A library for Python that provides automated refactoring capabilities, helping streamline the process of improving code quality.
+\end{itemize}
 \subsection{Products That Can Be Copied}
-\lips
+\begin{itemize}
+  \item \textbf{SonarQube:} An open-source platform designed for continuous inspection of code quality. It helps developers manage code quality and security by analyzing source code to identify potential issues.Its architecture and methods for detecting code smells could be adapted to focus specifically on energy efficiency. 
+\end{itemize}
 
 \section{New Problems}
 \subsection{Effects on the Current Environment}
```
Comment (Sevhena) [inline docs/SRS/SRS.tex:380]: I think we talked about how readability might become an issue right?

**Comment by Sevhena** [comment]
Also delete the pdf, there is no need to push pdfs, I'll add it to the gitignore in a pr at some point

File: docs/SRS/SRS.tex (hunk 1 above)
Comment (mmyaaaaa) [reply docs/SRS/SRS.tex:215]: we can talk about it in class. Someone has to install it at some point this is an easy screen record.

File: docs/SRS/SRS.tex (hunk 2 above)
Comment (mmyaaaaa) [reply docs/SRS/SRS.tex:380]: this question is asking for solutions out there in the market that are similar. whats wrong with it?

**Review by Sevhena — APPROVED** [review]
LGTM

**Pull request merged**